### PR TITLE
docs: Update to version 3.1.0 [pt-br]

### PR DESCRIPTION
**Pull request by periscuelo**
I made one update to be equal to english version explanation.
@ktsn Can you review / merge it?

Thanks

### CODE REVIEW

**Comment by kazupon** [comment]
/ping @ErickPetru
Could you review this PR? 🙏

File: docs/ptbr/api/README.md [hunk 1]
```diff
@@ -116,6 +116,19 @@ const store = new Vuex.Store({ ...options })
 
   [Detalhes](../guide/strict.md)
 
+### devtools
+
+- type: `Boolean`
+
+  Ative ou desative o _devtools_ para uma determinada instância vuex. Passar _false_ para a instância diz ao _store_ Vuex para não aderir ao plugin devtools. Útil para quando se tem vários _stores_ em uma _single page_.
```
Comment (ErickPetru) [inline docs/ptbr/api/README.md]:
```suggestion
  Ative ou desative as ferramentas de desenvolvedor para uma determinada instância vuex. Passar _false_ à instância diz ao _store_ Vuex para não se integrar ao _devtools_. Útil para quando se tem vários _stores_ em uma _single page_.
```

File: docs/ptbr/api/README.md [hunk 2]
```diff
@@ -194,6 +207,21 @@ const store = new Vuex.Store({ ...options })
 
   Para cancelar a assinatura, chame a função _unsubscribe_ retornada.
 
+  > Novo na 3.1.0
```
Comment (ErickPetru) [inline docs/ptbr/api/README.md]:
```suggestion
  > Novo em 3.1.0
```

File: docs/ptbr/guide/modules.md [hunk 3]
```diff
@@ -307,8 +307,6 @@ Você também pode remover um módulo dinamicamente registrado com o `store.unre
 
 Se usarmos um objeto simples para declarar o estado do módulo, esse objeto de estado será compartilhado por referência e causará poluição entre estados de _store_/módulo quando ele sofrer uma mutação.
 
-This is actually the exact same problem with `data` inside Vue components. So the solution is also the same - use a function for declaring module state (supported in 2.3.0+):
-
 Este é exatamente o mesmo problema com _data_ dentro dos componentes do Vue. Então a solução também é a mesma - use uma função para declarar o estado do módulo (suportado em 2.3.0+):
```
Comment (ErickPetru) [inline docs/ptbr/guide/modules.md]:
```suggestion
Este é exatamente o mesmo problema com `data` dentro dos componentes Vue. Então, a solução também é a mesma - use uma função para declarar o estado do módulo (suportado em 2.3.0+):
```

**Comment by periscuelo** [comment]
Thanks @ErickPetru. It's ready now.

**Comment by kazupon** [comment]
@ErickPetru
Thanks reviewing !

**Pull request merged**